app/views/metadata_presenter/analytics/analytics.html.erb

```diff
@@ -1,7 +1,11 @@
+<%# MoJ Forms Google Analytic Reporting %>
```

Comment:
Developer (template level) comment intentional because I'm already doing the HTML level one inside the app/views/metadata_presenter/analytics/_global_analytics.html.erb template. This one, like the `<%# Analytic reporting set by form owners %>` that also features in this partial, is an attempt to avoid developer confusion over what the partial is doing (and calling).